net-wireless/linssid/metadata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<upstream>
+		<bugs-to>https://sourceforge.net/p/linssid/discussion/bugs/</bugs-to>
+		<remote-id type="sourceforge">linssid</remote-id>
+	</upstream>
```

Comment:
like you did with other one, please fix ordering here, move upstream to the bottom.

Reply:
Yep, will do.